src/postamble.js

```diff
@@ -316,6 +316,10 @@ Module['exit'] = Module.exit = exit;
 var abortDecorators = [];
 
 function abort(what) {
+  if (Module['onRuntimeInitializeFailed']) {
```

Comment:
Hm, yeah, I wouldn't have any problem with that. As long as keeping my change of calling `abort` in the `WebAssembly.instantiate` catch isn't an issue, it'd be practically the same for my purposes.